machine_configuration/mco-coreos-layering.adoc

```diff
@@ -65,8 +65,8 @@ containerfileArch: noarch
 content: |-
   FROM configs AS final
   #Install hotfix rpm
-  RUN rpm-ostree override replace https://example.com/myrepo/haproxy-1.0.16-5.el8.src.rpm && \
-      rpm-ostree cleanup -m && \
+  RUN dnf swap haproxy https://example.com/myrepo/haproxy-1.0.16-5.el8.src.rpm && \
```

Comment:
Hmm, I think `dnf install` is more canonical for this. I.e. `dnf install -y https://...`. `dnf swap` should work too, but I think it's more used for situations where the package names to remove/install are different.